Xamarin.Essentials/Connectivity/Connectivity.ios.cs

```diff
@@ -13,6 +14,11 @@ static void StartListeners()
             listener.ReachabilityChanged += OnConnectivityChanged;
         }
 
+        static SignalStrength PlatformSignalStrength()
+        {
+            return (SignalStrength)(int)(new NEHotspotNetwork().SignalStrength * 3) + 1;
```

Comment:
I don't think a case like this is good, i would prefer to have a switch case. Same with Android above.

Reply:
done